modules/lang-painless/src/main/java/org/elasticsearch/painless/node/SFunction.java

```diff
@@ -168,7 +174,16 @@ void write(MethodWriter function) {
         }
     }
     
-    String getStaticHandleFieldName() {
+    private String getStaticHandleFieldName() {
         return Def.getUserFunctionHandleFieldName(name, parameters.size());
```

Comment:
I think you can remove that method now and inline the body above. It was just added to make the code in `SSource` not rely on internals here.